boards/arm/chester_nrf52840/chester_nrf52840.dts

```diff
@@ -31,40 +33,79 @@
 		btn_ext: btn_ext {
 			gpios = <&gpio0 26 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
 		};
+
+		tamper_key: tamper_key {
+			status = "disabled";
+			gpios = <&gpio0 12 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+		};
 	};
 
-	led_rgy {
+	gpio_leds {
 		compatible = "gpio-leds";
 
-		led_r: led_r {
-			gpios = <&gpio1 13 GPIO_ACTIVE_HIGH>;
-		};
-
 		led_g: led_g {
 			gpios = <&gpio1 11 GPIO_ACTIVE_HIGH>;
 		};
 
 		led_y: led_y {
 			gpios = <&gpio1 12 GPIO_ACTIVE_HIGH>;
 		};
-	};
 
-	led_ext {
-		compatible = "gpio-leds";
+		led_r: led_r {
```

Comment:
I suggest keeping the consistency with PWM LED and using the name `gpio_led_r`, etc. What do you think?

Reply:
don't know. The idea is to get closer to mainline. At least I didn't see anyone using this.

Reply:
OK